pages/[slug]/dashboard.tsx

```diff
@@ -60,7 +54,7 @@ const MemberRow = ({ member, user }: { member: Member; user: Member }) => {
 
   return (
     <li>
-      {member.email_address} ({member.status})
+      [{isAdmin(member) ? "admin" : "member"}] {member.email_address} ({member.status })
```

Comment:
same here - should this be in a different PR?

Reply:
🥺